src/nvim/sign.c

```diff
@@ -0,0 +1,1746 @@
+// This is an open source non-commercial project. Dear PVS-Studio, please check
+// it. PVS-Studio Static Code Analyzer for C, C++ and C#: http://www.viva64.com
+
+//
+// sign.c: functions for managing with signs
+//
+
+
+#include "nvim/vim.h"
+#include "nvim/sign.h"
+#include "nvim/ascii.h"
+#include "nvim/buffer.h"
+#include "nvim/charset.h"
+#include "nvim/cursor.h"
+#include "nvim/ex_docmd.h"
+#include "nvim/edit.h"
+#include "nvim/fold.h"
+#include "nvim/move.h"
+#include "nvim/screen.h"
+#include "nvim/syntax.h"
+
+/// Struct to hold the sign properties.
+typedef struct sign sign_T;
+
+struct sign
+{
+    sign_T      *sn_next;       // next sign in list
+    int         sn_typenr;      // type number of sign
+    char_u      *sn_name;       // name of sign
+    char_u      *sn_icon;       // name of pixmap
+# ifdef FEAT_SIGN_ICONS
```

Comment:
Where is this defined? I can't find `#define FEAT_SIGN_ICONS` in the codebase so I assume this is dead code.

Reply:
Yep, we should remove the guard